### PR TITLE
settings section

### PR DESCRIPTION
**Pull request by stormcloud266**
Adds card for settings, minor link style updates. each individual settings section will be stacked

Screenshot 2024-09-13 at 6 54 13 PM

### CODE REVIEW

**Comment by stormcloud266** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/713?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#713** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/713?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#711** <a href="https://app.graphite.dev/github/pr/0xsend/sendapp/711?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `dev`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @stormcloud266 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/app/features/account/settings/SettingsNavLink.tsx [hunk 1]
```diff
@@ -17,33 +17,32 @@ export function SettingsNavLink({
...
         pl="$4"
+        fontSize={'$5'}
       >
         {text}
       </Paragraph>
       {isActiveRoute && !sheet.open && (
         <Separator
           vertical
-          borderColor="$greenBackground"
+          borderColor="$primary"
```
Comment (youngkidwarrior) [inline packages/app/features/account/settings/SettingsNavLink.tsx:44]: Fun fact:

The `color12` and `color0` values are the same for every theme. They are our foreground and background colors a.k.a black and white

So you can do this with themes by using `<Separator theme="green" $theme-light={{boc:"$color12"}}/>`

**Review by youngkidwarrior — APPROVED** [review]
lgtm just a nit

**Review by youngkidwarrior — APPROVED** [review]

**Comment by youngkidwarrior** [comment]
### Merge activity

* **Sep 14, 2:43 AM EDT**: @youngkidwarrior started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/0xsend/sendapp/713).

**Pull request merged**